### DISCUSSION TITLE
How do I split a line, and how do I make one?

### DISCUSSION THREAD

**Discussion by rhavin** [fontforge/fontforge#5403] [Q&A] [answered]
I have two point. If i click them, and select "point>make line", I get a line.

If I have a line between two points, how do I remove the line without removing the points?

Also, I often cant get a straight line, even if I make both points corners. How can I remove the handles without removing the points?

**Comment by rybak** [comment]
> If I have a line between two points, how do I remove the line without removing the points?

I think that the only way to do it is to:

1. Zoom in really close on the first point
2. Cut with Knife tool perpendicular to the line at the point exactly
3. Zoom in really close on the second point
4. Cut with "Knife" tool (shortcut <kbd>K</kbd>) perpendicular to the line at the point exactly
5. Now you've got a line which is cut off from the other lines/contours/etc
6. Select "Pointer" tool (<kbd>V</kbd>)
7. Drag the new piece of line away from two points
8. Press <kbd>Delete</kbd> key

**Comment by skef** [comment]
> If I have a line between two points, how do I remove the line without removing the points?

Have you tried:

1. Use the knife tool to cut the line somewhere between the two points
2. Select the points you just created with the knife tool by dragging a rectangle around them using the pointer tool
3. Press the delete key

?

> How can I remove the handles without removing the points?

The easiest way to do this is typically to select the two points in question, right click on the background, and pick "make line" from the menu. "Make Line" is also in the Point menu up top.

**Comment by iorsh** [comment, marked as answer]
Building on @skef's proposal, you can add a single point on the target segment, and then deleting it would delete the segment together with the newly created point.

**Reply by rhavin** [reply]
That is a completely stupid way to do it but seemingly the only one useful :D
